### ISSUE TITLE
Thunderbird notifications for spam messages

### ISSUE THREAD

**Issue by noahheck** [mkiol/GNotifier#166]
Recently (within the last week), I've started receiving notifications from Thunderbird when receiving messages that are marked as spam. This is different from the behavior from before where notifications were not shown for spam messages. When looking for the cause of the issue, I see that my GNotifier was updated on May 3, which coincides with the timeline of my issue.

I'm using Thunderbird 45.8.0 on Linux Mint 18.1.

**Comment by mkiol** [comment]
Thank you for reporting this. In the latest update I changed code responsible for new messages detection and probably mess something up. I will troubleshoot this.

**Comment by noahheck** [comment]
Great! Please let me know if I can help in some way :)

**Comment by lightonflux** [comment]
Does it look like this? 
![bildschirmfoto von 2017-05-08 23-43-02](https://cloud.githubusercontent.com/assets/1377943/25830927/c709e1e8-3460-11e7-84f0-d09662326cce.png)


I am not sure what this is. If these notifications are mails that are marked as junk and moved to spam or if these are just broken notifications. 

If it doesn't look like this for you then i open a new issue.

**Comment by noahheck** [comment]
Looks to be the same. 
![spam_message](https://cloud.githubusercontent.com/assets/4154306/25831321/6c7255a2-3420-11e7-8147-f351143cca96.png)

**Comment by mkiol** [comment]
I can confirm this bug. I'm working on the fix.

**Comment by mkiol** [comment]
Please try the [latest (1.10.3) XPI](https://github.com/mkiol/GNotifier/tree/master/xpi). The issue with "empty" notifications should be resolved. I would be very grateful for testing it and feedback. Thanks!

**Comment by noahheck** [comment]
I have disabled all other addons, and installed the latest xpi (1.10.3) and am still receiving the notifications for spam messages.

**Comment by mkiol** [comment]
Thank you. My "problem" is that I hardly ever receive spam messages and it makes me hard to do proper tests.

Please try [1.10.4](https://github.com/mkiol/GNotifier/tree/master/xpi).

**Comment by noahheck** [comment]
Ha! First world problems. I'll trade you ;)

When I got in this morning, I discovered that Thunderbird on Linux Mint has been updated to 52.1.1 if that will affect testing.

I installed the 1.10.4 xpi and it seems to be functioning as expected now (i.e. not getting notifications for spam messages but still getting messages for valid ones).

Thank you so much! Let me know if there's anything else I can do.
